networking/change-block-size.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Change IP pool block size
+description: Expand or shrink the IP pool block size to efficiently manage IP pool addresses. 
+---
+
+### Big picture
+
+Change the IP pool block size to efficiently manage IP pool addresses. 
+
+### Value
+
+Because the IP pool block size is configured during install and cannot be edited, you must follow the steps outlined in this article to minimize pod connectivity disruption. 
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **IPPool** resource with `blockSize` field
+
+### Concepts
+
+#### Best practices
+
+- **Create IP pools within the same cluster CIDR** 
```

Comment:
While true, I don't think this is relevant to changing the block size, right?

Reply:
Deleted.